sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/projects/_version.py

```diff
@@ -0,0 +1 @@
+from .._version.py import VERSION
```

Comment:
Could you please update the underlying package version and add an update to the change log?

Reply:
what should be the version number?

Reply:
I think in the chat we decided 1.1.0b1 :)